### PR TITLE
[RDKDEV-721] XCast limit payload to avoid overflow

### PR DESCRIPTION
**Pull request by dawid-trendota-red**
Fix for

https://jira.rdkcentral.com/jira/browse/RDKDEV-721

### CODE REVIEW

**Comment by pradeeptakdas** [comment]
Internal ticket for tracking https://ccp.sys.comcast.net/browse/RDKCOM-3724

File: XCast/XCastCommon.h [hunk 1]
```diff
@@ -24,8 +24,9 @@
 #define __XCAST_COMMON_H__
 /*
  * The maximum DIAL payload accepted per the DIAL 1.6.1 specification.
+ * payload from xdialserver is URI encoded
  */
-#define DIAL_MAX_PAYLOAD (4096)
+#define DIAL_MAX_PAYLOAD (4096*3)
```
Comment (apatel859) [inline XCast/XCastCommon.h:29]: max payload allwayed is 4kb why multiplied by 3 which makes it 12kb?

Comment (dawid-trendota-red) [reply XCast/XCastCommon.h:29]: This payload here is base64 encoded, so it can require up to 3 times more memory then unencoded payload.

**Comment by pradeeptakdas** [comment]
Hi @apatel859 
Can you please review and merge this PR?

**Pull request merged**